From 7e7912903946f9d51e1c482bf6e1e717ca738509 Mon Sep 17 00:00:00 2001
From: Nvertigo <nvertigo67@gmail.com>
Date: Tue, 12 Jun 2018 13:47:36 +0200
Subject: [PATCH] Revert "Gallery2: set module privileged in Android.mk to
 delete photos on SD card."

This reverts commit d22a16065092b77febc6d0c0e0c388ad20a918bc.

  Making Gallery2 previleged breaks "Settings->User->(Crop|Take) Picture",
  because priv_app is denied to open system_data_file by a neverallow
  rule from LineageOS/android_system_sepolicy.

  For non previleged apps there is a dedicated rule
  https://github.com/LineageOS/android_system_sepolicy/blob/lineage-15.1/private/untrusted_app_all.te#L47
  to allow read and write system app data files passed over by Binder.

  The reason for the original commit has been issues deleting photos on SD
  card. May be there's an other solution, or it has already been solved,
  because user apps (like GooglePhotos installed as user app from
  playstore) are able to delete pictures from SD Card.

  Anyway:
  Making Gallery2 previleged leads to unrsolvable sepolicy denials.

Change-Id: Ia3ee43146064c6258026fcb5b6ffe909446f1184
---
 Android.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 723ed7eeb..6184fe1fe 100644
--- a/Android.mk
+++ b/Android.mk
@@ -39,8 +39,6 @@ LOCAL_STATIC_JAVA_AAR_LIBRARIES := \
 
 LOCAL_PACKAGE_NAME := Gallery2
 
-LOCAL_PRIVILEGED_MODULE := true
-
 LOCAL_OVERRIDES_PACKAGES := Gallery Gallery3D GalleryNew3D
 
 LOCAL_AAPT_FLAGS += --rename-manifest-package com.android.gallery3d
-- 
2.17.1

